docs/design/integration_nova.adoc

```diff
@@ -20,16 +20,194 @@ We would want to integrate with Nova to support the same set of user operations,
+|show_network
+|get
+|net_id
+|dns_domain
+|/v2.0/networks/{net_id}
+|/v4/{project_id}/vpcs/{vpcid}
+
+|show_network
+|get
+|net_id
+|segments
+|/v2.0/networks/{net_id}
+|/v4/{project_id}/vpcs/{vpcid}
+
+|list_network
+|get
+|
+|
+|/v2.0/networks
+|/project/{project_id}/vpcs
+
+|list_network
+|get
+|net_ids
+|
+|/v2.0/networks
+|/project/{project_id}/vpcs
+
+|list_network
+|get
+|tenant_id: project_id, shared: False
+|
+|/v2.0/networks
+|/project/{project_id}/vpcs(no support)
+
+|list_network
+|get
+|tenant_id: project_id, shared: False, admin_state_up: True
+|
+|/v2.0/networks
+|/project/{project_id}/vpcs(no support)
+
+|list_network
+|get
+|shared: True
+|
+|/v2.0/networks
+|/project/{project_id}/vpcs(no support)
+
+|list_subnets
+|get
+|id: [subnet_id]
+|
+|/v2.0/networks
+|/project/{project_id}/subnets
+
+|show_port
+|get
+|
+|binding_profile, network_id
+|/v2.0/ports/{port_id}
+|/project/{project_id}/ports/{port_id}
+
+|show_port
+|get
+|
+|binding:vnic_type,network_id,binding_profile,resource_request
+|/v2.0/ports/{port_id}
+|/project/{project_id}/ports/{port_id}
+
+|list_ports
+|get
+|network_id: net_id,  device_owner: network_dhcp
+|
+|/v2.0/ports
+|/project/{project_id}/ports(no support)
+
+|list_ports
+|get
+|device_id: instance.uuid, tenant_id: project_id
+|
+|/v2.0/ports
+|/project/{project_id}/ports(no support)
+
+|list_ports
+|get
+|network_id: net_id, fixed_ips: ip_addrs
+|device_id
+|/v2.0/ports
+|/project/{project_id}/ports(no support)
+
+|create_port
+|post
+|port: {device_id: instance.uuid, fixed_ips: fixed_ip, network_id: net_id, admin_state_up: True, tenant_id: project_id, security——groups:{}}
+|
+|/v2.0/ports
+|/project/{project_id}/ports
+
+|update_port
+|put
+|port: {device_id: '', device_owner: '', 'binding:host_id': None, 'binding:profile': {}}
+|
+|/v2.0/ports/{port_id}
+|/project/{project_id}/ports/{port_id}
+
+|update_port
+|put
+|port: {dns_name: ''}
+|
+|/v2.0/ports/{port_id}
+|/project/{project_id}/ports/{port_id}
+
+|delete_port
+|delete
+|
+|
+|/v2.0/ports/{port_id}
+|/project/{project_id}/ports/{port_id}
+
+|list_floatingips
+|get
+|fixed_ip_address: fixed_ip, port_id: port_id
+|
+|/v2.0/floatingips
+|
+
+|list_floatingips
+|get
+|fixed_ip_address: fixed_ip
+|
+|/v2.0/floatingips
+|
+
+|show_quota
+|get
+|
+|
+|/v2.0/quotas/{project_id}
+|
+
+|list_extensions
+|get
+|
+|return example:{'extensions':{{updated: 2017-07-17T10:00:00-00：00， name:port_binding_extended, links:[], alias: binding-extended, description: Expose port binding of a virtual port to external application}}}
+|/v2.0/extensions
+|
+|===
+
 === VM Creation Workflow
 image::images/vm_create.png["VM creation workflow", width=1024, link="images/vm_create.png"]
 
 == Required Changes
+https://github.com/openstack/python-neutronclient[neutronclient project]
 
-TBD
+Nova use python-neutronclient to call neutron apis.Only need to change neutronclient/v2.0/client:Client class.
 
 == Integration Proposal
 
-TBD
+There are two ways to integration with nova:
```

Comment:
Assuming that Alcor APIs is 100% compatible to Neutron, do we still need make changes to Nova? For example, Nova would need to know the server url and port of a new Alcor cluster. It is likely a Nova configuration. Could you list those configuration change as well?

Reply:
@Gzure Could you please address the comments based on the discussion in today's open-source meeting? Thank you.